inst/text/gitignore.txt

```diff
@@ -77,3 +77,10 @@ po/*~
 
 # RStudio Connect folder
 rsconnect/
+
+# Images in root directory
+/*.jpg
+/*.jpeg
+/*.png
+/*.gif
+/*.tiff
```

Comment:
Add `.bmp` and `.svg` and make the ignores case insensitive.
```suggestion
/*.[jJ][pP][gG]
/*.[jJ][pP][eE][gG]
/*.[pP][nN][gG]
/*.[gG][iI][fF]
/*.[tT][iI][fF][fF]
/*.[bB][mM][pP]
/*.[sS][vV][gG]
```

Reply:
https://github.com/Public-Health-Scotland/phstemplates/blob/4a7b87232ab5fa208dfacd7b80c162e50fceaf5f/inst/text/gitignore.txt#L41 Already there!

If you do spot / think of other stuff that needs adding it would make sense to make a separate PR though (so that this one is focused on the image addition).